bodhi-server/bodhi-server.spec

```diff
@@ -35,6 +35,10 @@ BuildRequires:  createrepo_c
 BuildRequires:  skopeo
 BuildRequires:  dnf
 
+%if 0%{?fedora} && 0%{?fedora} >= 41
+BuildRequires:  python3-libdnf5 >= 5.2
+%endif
+
```

Comment:
Instead of hardcoding the version here, pass `+x libdnf5` to `%pyproject_buildrequires`.